### PR TITLE
Log error when slides fail to load

### PR DESCRIPTION
**Pull request by andrecardoso**

### CODE REVIEW

**Comment by andrecardoso** [comment]
[(2) Log errors when loading the iframe fails](https://trello.com/c/z8t0zq6s/85-2-log-errors-when-loading-the-iframe-fails)

File: src/rise-slides.js [hunk 1]
```diff
@@ -49,19 +51,24 @@ export default class RiseSlides extends PolymerElement {
...
     } else {
-      window.addEventListener( "rise-components-ready", () => this._init(), { once: true });
+      window.addEventListener( "rise-components-ready", () => this._init(), {once: true});
     }
   }
 
   _init() {
     this.addEventListener(RiseSlides.EVENT_START, this._handleStart, {once: true});
     this._sendEvent(RiseSlides.EVENT_CONFIGURED);
+    this._loadTimer = setTimeout(() => this.log("error", "loading slides timeout"), 10000);
```
Comment (olegrise) [inline src/rise-slides.js]: @andrecardoso do you think we should log error when component is previewed in Editor?

File: src/rise-slides.js [hunk 2]
```diff
@@ -49,19 +51,24 @@ export default class RiseSlides extends PolymerElement {
     this._started = false;
   }
 
+  get iframe() {
```
Comment (olegrise) [inline src/rise-slides.js]: @andrecardoso where is this used?

Comment (andrecardoso) [reply src/rise-slides.js]: Nowhere, I'll delete it. Thanks.

File: src/rise-slides.js (hunk 1 above)
Comment (andrecardoso) [reply src/rise-slides.js]: I think we should avoid it. I'll change it. Thanks.

**Pull request merged**